optimizer_wrapper.rb

```diff
@@ -363,78 +374,84 @@ def self.split_independent_vrp_by_skills(vrp)
         independent_skills[set_index] += b_set
       }
     }
-
-    # Original skills are retrieved
-    independant_skill_sets = independent_skills.map{ |index_set|
+    # Independent skill sets : Original skills are retrieved
+    independent_skills.map{ |index_set|
       index_set.collect{ |index| mission_skills[index] }
     }
+  end
 
-    unused_vehicles_indices = (0..vrp.vehicles.size - 1).to_a
-    independent_vrps = independant_skill_sets.collect{ |skills_set|
+  def self.build_independent_vrps(vrp, skill_sets, unused_vehicle_indices, skill_vehicle_ids, skill_service_ids)
```

Comment:
Maybe _unused_vehicle_indices_ could be initialized here to reduce this function number of parameters.

Reply:
As it is updated during through the generation of the multiple independent vrps, we must initialize it before the call this method.